src/adafruit_blinka/microcontroller/am335x/pin.py

```diff
@@ -344,9 +344,4 @@ def value(self, val=None):
     (2, I2C2_SCL, I2C2_SDA),
 )
 
-PWM1 = P1_36
-PWM2 = P1_33
-PWM3 = P2_1
-PWM4 = P2_3
-
 pwmOuts = ( ((0, 0), PWM1), ((0, 1), PWM2), ((2, 0), PWM3), ((4, 1), PWM4) )
```

Comment:
For this line, since PWM1-PWM4 haven't been defined, what is is using for those values?

Reply:
Thanks for the review.  The PWM1-PWM4 are moved in commit https://github.com/pdp7/Adafruit_Blinka/commit/24df77e78c7f304d55f3ec6b8e983e42340e648a# from `src/adafruit_blinka/microcontroller/am335x/pin.py` to:
* `src/adafruit_blinka/board/beaglebone_black.py`
* `src/adafruit_blinka/board/beaglebone_pocketbeagle.py`
where they are correctly defined for PocketBeagle using its P1/P2 headers and the BeagleBone using its P8/P9 headers

Does that make sense?

Reply:
The order doesn't make sense. The Board file is loaded first, which loads the Pins file via an import at the top which defines the pwmOuts variable. Later, lower in the board file, which is also out of scope for the Pins file, PWM1-4 are defined.

Reply:
Ok, thanks.  I'll rework the way I'm defining different PWM1-4 pins for PocketBeagle vs. BeagleBone